### PR TITLE
nodePackage.firebase-tools: init at 3.9.1

### PR DESCRIPTION
**Pull request by kirelagin**

### CODE REVIEW

**Comment by kirelagin** [comment]
I should probably note that I am completely new to all this nodejs business (both nix-related and in general). I’m a little bit worried that this commits also updates lots of other packages and I don’t know how bad that is, but, well, I followed the README.

Speaking about firebase-tools, I verified that it properly communicates with the server, lists projects and successfully deployed my website.

File: pkgs/development/node-packages/node-packages.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
 , "eslint"
 , "emojione"
 , "fetch-bower"
+, "firebase-tools"
```
Comment (Mic92) [inline pkgs/development/node-packages/node-packages.json]: an alias in `all-packages.nix` would be nice:

```
  firebase-tools = nodePackages.firebase-tools;
```

**Comment by kirelagin** [comment]
Are node packages dead? That’s sad 😞.

**Comment by kirelagin** [comment]
Hiiiii ✋

**Comment by wizzup** [comment]
Seem like nobody have solution for `generate.sh` mess with these two files,

    pkgs/development/node-packages/node-packages-v4.nix
    pkgs/development/node-packages/node-packages-v6.nix

#27952  #23771 #25770

**Comment by c0bw3b** [comment]
Closing this for now because this is way too much LOC to add to nixpkgs for a single package. Plus v3.9.1 is already obsoleted upstream anyway.
But feel free to reopen or come back with a new PR if you work on this again. :)

**Comment by Mic92** [comment]
The lines of code might be not only only accounted for the tool itself but also updating other packages.

**Comment by kirelagin** [comment]
The fact that I get back to this PR relatively rarely doesn’t meen that I don’t want `firebase-tools` in nixpkgs. I still do. If all the PRs adding node expressions are going to be closed as “too much LOC” then I’m not sure how to proceed.

**Comment by wizzup** [comment]
I don't have authority to decide how to fix `node-packages-v*.nix` but if you want to try reducing the diff, try `checkout` latest version of these files from `master` and rerun `generate.sh`.

**Pull request closed without merging**